test/agent_helper.rb

```diff
@@ -871,7 +871,7 @@ def load_cross_agent_test(name)
   test_file_path = File.join(cross_agent_tests_dir, "#{name}.json")
   data = File.read(test_file_path)
   data.gsub!('callCount', 'call_count')
-  data = JSON.load(data)
+  data = JSON.parse(data)
```

Comment:
`load` takes a file or a JSON string,`parse` takes a JSON string. We only pass a string. 👍